README/ReleaseNotes/v626/index.md

```diff
@@ -406,6 +406,20 @@ Before 6.26, it was possible to still use the `RooMinuit` by passing the `Minimi
 The class version of `RooAbsArg` was incremented from 7 to 8 in this release. In some circumstances, this can cause warnings in `TStreamerInfo` for classes inheriting from `RooAbsArg` when reading
 older RooFit models from a file. These warnings are harmless and can be avoided by incrementing also the class version of the inheriting class.
 
+### Compile-time protection against creating empty `RooCmdArg`s from strings
+
+The implicit [RooCmdArg](https://root.cern.ch/doc/v626/classRooCmdArg.html) constructor from `const char*` was removed to avoid the accidental construction of meaningless RooCmdArgs that only have a name but no payload.
+This causes new compiler errors in your code if you pass a string instead of a RooCmdArg to various RooFit functions, such as [RooAbsPdf::fitTo()](https://root.cern.ch/doc/v626/classRooAbsPdf.html#a5f79f16f4a26a19c9e66fb5c080f59c5).
```

Comment:
```suggestion
This causes new compiler errors in your code if you pass a string instead of a RooCmdArg to various RooFit functions, such as [RooAbsPdf::fitTo()](https://root.cern/doc/v626/classRooAbsPdf.html#a5f79f16f4a26a19c9e66fb5c080f59c5).
```